org.eclipse.lyo.oslc4j.adaptormodel/model/vocabulary.ecore

```diff
@@ -8,12 +8,7 @@
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="Vocabulary">
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="namespaceURI" lowerBound="1"
-        eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString">
```

Comment:
what's this change about?

Reply:
Just give me 2 minutes to finish writing :-)

Reply:
I had problems compiling/generating java from the ecore files due to this.
I guess new version of EMF became more picky, causing errors that we did not have before.